tableauserverclient/server/pager.py

```diff
@@ -4,12 +4,23 @@
 
 class Pager(object):
     """
-    Generator that takes an endpoint with `.get` and lazily loads items from Server.
-    Supports all `RequestOptions` including starting on any page.
+    Generator that takes an endpoint (top level endpoints with `.get)` and lazily loads items from Server.
+    Supports all `RequestOptions` including starting on any page. Also used by models to load sub-models
+    (users in a group, views in a workbook, etc) by passing a different endpoint.
+
+    Will loop over anything that returns (List[ModelItem], PaginationItem).
     """
 
     def __init__(self, endpoint, request_opts=None):
-        self._endpoint = endpoint.get
+
+        if hasattr(endpoint, 'get'):
+            # The simpliest case is to take an Endpoint and call its get
+            self._endpoint = endpoint.get
+        else:
```

Comment:
Elif callable(...):
   ...
Else:
   raise ...

(Sorry I'm on my phone, didn't bring a computer with me to California)